### DISCUSSION TITLE
Need help with .env

### DISCUSSION THREAD

**Discussion by MikeAlbert4Real** [jef/streetmerchant#1144] [Help]
Some help if possible
I been using the script to try and find get my hand on a next gen console (ps5 and Xbox series x). I’m not a coding guy but am computer savvy but feel like I’m doing something wrong. I get no hits when I know there have been drops. I’m asking if someone could possibly help me out with creating me an .env file that works. I’m looking in the us stores on all the retailers that would carry it. Can someone please help.

**Comment by strager** [comment]
Is streetmerchant reporting "OUT OF STOCK", or is it reporting nothing at all?

Can you share the full output of `npm run start`, after it's been running for a few minutes?

Can you share your existing .env file?

**Comment by MikeAlbert4Real** [comment]
it just reports out of stock, I just recieved an env file from a friend but I keep getting these "cound not find files" messages when i start up the script
![Screenshot (7)](https://user-images.githubusercontent.com/75456294/101118013-41408700-35b6-11eb-8be4-751c2fb965ad.png)

**Comment by KaleRoberts** [comment]
Yeah I would imagine it's an unsupported configuration in your .env file. We'd need to what your actual file looks like to know for sure.

**Comment by MikeAlbert4Real** [comment]
MAX_PRICE_SERIES_RX6800=""
MAX_PRICE_SERIES_RX6800XT=""
MAX_PRICE_SERIES_RX6900XT=""
MAX_PRICE_SERIES_RYZEN5600=""
MAX_PRICE_SERIES_RYZEN5800=""
MAX_PRICE_SERIES_RYZEN5900=""
MAX_PRICE_SERIES_RYZEN5950=""
MAX_PRICE_SERIES_SONYPS5C=""
MAX_PRICE_SERIES_SONYPS5DE="1000"
MICROCENTER_LOCATION=""
MQTT_BROKER_ADDRESS=""
MQTT_BROKER_PORT=""
MQTT_CLIENT_ID=""
MQTT_PASSWORD=""
MQTT_QOS=""
MQTT_TOPIC=""
MQTT_USERNAME=""
NVIDIA_ADD_TO_CART_ATTEMPTS=""
NVIDIA_SESSION_TTL=""
OPEN_BROWSER="true"
PAGE_BACKOFF_MIN=""
PAGE_BACKOFF_MAX=""
PAGE_SLEEP_MIN=""
PAGE_SLEEP_MAX=""
PAGE_TIMEOUT=""
PAGERDUTY_INTEGRATION_KEY=""
PAGERDUTY_SEVERITY=""
PHILIPS_HUE_API_KEY=""
PHILIPS_HUE_CLOUD_ACCESS_TOKEN=""
PHILIPS_HUE_CLOUD_CLIENT_ID=""
PHILIPS_HUE_CLOUD_CLIENT_SECRET=""
PHILIPS_HUE_CLOUD_REFRESH_TOKEN=""
PHILIPS_HUE_LAN_BRIDGE_IP=""
PHILIPS_HUE_LIGHT_COLOR=""
PHILIPS_HUE_LIGHT_IDS=""
PHILIPS_HUE_LIGHT_PATTERN=""
PHONE_CARRIER=""
PHONE_NUMBER=""
PLAY_SOUND="Wake up.wav"
PROXY_ADDRESS=""
PROXY_PROTOCOL=""
PROXY_PORT=""
PUSHBULLET=""
PUSHOVER_TOKEN=""
PUSHOVER_USER=""
PUSHOVER_PRIORITY=""
SCREENSHOT=""
SHOW_ONLY_BRANDS=""
SHOW_ONLY_MODELS=""
SHOW_ONLY_SERIES="sonyps5c,sonyps5de"
SLACK_CHANNEL=""
SLACK_TOKEN=""
SMTP_ADDRESS=""
SMTP_PORT=""
STORES="amazon,bestbuy,target,walmart,officedepot,antonline"
TELEGRAM_ACCESS_TOKEN=""
TELEGRAM_CHAT_ID=""
TWILIO_ACCOUNT_SID=""
TWILIO_AUTH_TOKEN=""
TWILIO_FROM_NUMBER=""
TWILIO_TO_NUMBER=""
TWITCH_ACCESS_TOKEN=""
TWITCH_CHANNEL=""
TWITCH_CLIENT_ID=""
TWITCH_CLIENT_SECRET=""
TWITCH_REFRESH_TOKEN=""
TWITTER_ACCESS_TOKEN_KEY=""
TWITTER_ACCESS_TOKEN_SECRET=""
TWITTER_CONSUMER_KEY=""
TWITTER_CONSUMER_SECRET=""
TWITTER_TWEET_TAGS=""
USER_AGENT=""
WEB_PORT=""

**Comment by KaleRoberts** [comment]
I think I see what it is and it's probably the value you have here for 
PLAY_SOUND="Wake up.wav"

That needs to be a relative path to wherever you have that sound file. If you have that sound file pasted directly into the root of the project then you'll want to change that to the following:
PLAY_SOUND="./Wake_up.wav"
Probably should rename the file too like I have it there since I haven't dug into the code to see if spaces are ignored.

**Comment by MikeAlbert4Real** [comment]
Thank you so much for the reply and help. The PLAY_SOUND="Wake up.wav" actually does work, I tested the notification sound and did install the cmdmp3 binaries. I’ve gone through the .env file and double checked that all the variables were correct but still no luck.
